app/src/main/java/com/krachkovsky/it_anews/retrofit/NewsService.kt

```diff
@@ -0,0 +1,28 @@
+package com.krachkovsky.it_anews.retrofit
+
+import com.krachkovsky.it_anews.util.Constants.BASE_URL
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.create
+
+object NewsService {
+    private val retrofit by lazy(LazyThreadSafetyMode.NONE) { provideRetrofit() }
+    val newsApi by lazy(LazyThreadSafetyMode.NONE) {
+        retrofit.create<NewsApi>()
+    }
+
+    private fun provideRetrofit(): Retrofit {
+        val logging = HttpLoggingInterceptor()
+        logging.setLevel(HttpLoggingInterceptor.Level.BODY)
```

Comment:
по хорошему логи нужно ограничивать дебаг сборкой приложения
```suggestion
        if (BuildConfig.DEBUG) {
            val logging = HttpLoggingInterceptor()
            logging.setLevel(HttpLoggingInterceptor.Level.BODY)
        }
```